### PR TITLE
Test case for Special Page added

### PR DESCRIPTION
**Pull request by Parul4214**

### CODE REVIEW

File: src/test/java/com/naveenautomation/tests/SpecialsPageTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		myAccountPage = accountLoginPage.login("parul.verma421@gmail.com", "Parul4");
+
+	}
+
+	@Test
+	public void verifyUserIsAbleToAddItemFromSpecials() {
+		SpecialsPage specialPage = myAccountPage.clickOnSpeacialInExtras();
+		specialPage.clickOnAddToWishlist();
+
+		softAssert.assertEquals("Success: You have added Apple Cinema 30\" to your wish list!\n" + "×",
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/SpecialsPageTest.java:32]: use string methods and remove all unnecessary characters

File: src/test/java/com/naveenautomation/tests/SpecialsPageTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import com.naveenautomation.Base.TestBase;
+import com.naveenautomation.Pages.AccountLoginPage;
+import com.naveenautomation.Pages.MyAccountPage;
+import com.naveenautomation.Pages.SpecialsPage;
+
+public class SpecialsPageTest extends TestBase {
+
+	SoftAssert softAssert = new SoftAssert();
+	AccountLoginPage accountLoginPage;
```
Comment (PivotITAcademy) [inline src/test/java/com/naveenautomation/tests/SpecialsPageTest.java:16]: where are access modifiers?

**Review by PivotITAcademy — CHANGES_REQUESTED** [review]

File: src/test/java/com/naveenautomation/tests/SpecialsPageTest.java (hunk 2 above)
Comment (Parul4214) [reply src/test/java/com/naveenautomation/tests/SpecialsPageTest.java]: Code updated with access modifiers

File: src/test/java/com/naveenautomation/tests/SpecialsPageTest.java (hunk 1 above)
Comment (Parul4214) [reply src/test/java/com/naveenautomation/tests/SpecialsPageTest.java]: Code updated using String Method

**Review by PivotITAcademy — APPROVED** [review]
LGTM

**Pull request merged**